templates/zerver/emails/new_login/new_login_alert.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+<head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <title>Zulip</title>
+</head>
+<body>
+    <table width="80%" style="align:center; max-width:800px" align="center">
+        <tr>
+            <td>
+                <a href="{{ realm_uri }}/"><img style="max-height:75px; height:75px;"  height="75px" alt="Zulip" title="{{ _('Zulip') }}" src="{{ realm_uri }}/static/images/landing-page/zulip-header.png" /></a>
+                <h3 style="font-family:Arial; font-size:30px; margin: 5px 0px; color:#555">New Login To Zulip</h3>
+            </td>
+        </tr>
+        <tr>
+            <td style="font-size:16px; font-family:Helvetica;">
+                <p><b>Hello, {{ user.short_name | title }}.</b></p>
+                <p>This is a notification that a new login to your Zulip account has just occured.</p>
+                <p><b>Details:</b></p>
+                <blockquote>Account: {{ user.email }}<br />Date/Time: {{ device_info.login_time }}<br />Device: {{ device_info.device_browser | title }} on {{ device_info.device_os | title }}<br />IP Address: {{ device_info.device_ip }}</blockquote>
```

Comment:
Should there be line breaks in the HTML after each `<br />`?  I think that'd be more readable.  

Also see comments on the content below.